.env

```diff
@@ -41,3 +41,17 @@ APP_LOG_RECIPIENT_1=admin1@example.org
 APP_LOG_RECIPIENT_2=admin2@example.org
 APP_LOG_SUBJECT="[Wikisource Export]"
 EPUBCHECK_JAR="/usr/bin/epubcheck"
+# For production, REPLICAS_HOST should be *.web.db.svc.eqiad.wmflabs (or analytics instead of web)
+# For local environments, use 127.0.0.1 and host.docker.internal for docker.
+REPLICAS_HOST="127.0.0.1"
```

Comment:
Sorry, I got this wrong. In production we'll need a separate env variable for each host (s1 through s8). See my update to the ToolforgeBundle README https://github.com/wikimedia/ToolforgeBundle/pull/46

So here it should be:
```
REPLICAS_HOST_S1="127.0.0.1"
REPLICAS_HOST_S2="127.0.0.1"
REPLICAS_HOST_S3="127.0.0.1"
REPLICAS_HOST_S4="127.0.0.1"
REPLICAS_HOST_S5="127.0.0.1"
REPLICAS_HOST_S6="127.0.0.1"
REPLICAS_HOST_S7="127.0.0.1"
REPLICAS_HOST_S8="127.0.0.1"
```
also change doctrine.yaml accordingly.

Reply:
You didn't get it wrong. You figured this out before we did :) Thank you!